python313Packages.mwdblib: 4.5.0 -> 4.6.0

fabaff opened the pull request:
Diff: https://github.com/CERT-Polska/mwdblib/compare/v4.5.0...v4.6.0

Changelog: https://github.com/CERT-Polska/mwdblib/releases/tag/v4.6.0


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 446691`
Commit: `b222a1432d66a795d09ac3ec9968783bf837d0ad`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.karton-mwdb-reporter</li>
    <li>python312Packages.karton-mwdb-reporter.dist</li>
    <li>python312Packages.mwdblib</li>
    <li>python312Packages.mwdblib.dist</li>
    <li>python313Packages.karton-mwdb-reporter</li>
    <li>python313Packages.karton-mwdb-reporter.dist</li>
    <li>python313Packages.mwdblib</li>
    <li>python313Packages.mwdblib.dist</li>
  </ul>
</details>

mdaniels5757 reviewed (commented):
python-dateutil was dropped as a dependency in v4.0.0 (https://github.com/CERT-Polska/mwdblib/commit/e87a19f96eb691009fc8b2491fb3f7ed26c51d68). Otherwise LGTM.

fabaff committed:
python313Packages.mwdblib: 4.5.0 -> 4.6.0

Diff: CERT-Polska/mwdblib@v4.5.0...v4.6.0

Changelog: https://github.com/CERT-Polska/mwdblib/releases/tag/v4.6.0
